src/main/java/backend/Logic.java

```diff
@@ -119,6 +120,7 @@ public void refresh() {
                         .thenApply(models::addPending)
                         .thenRun(this::updateUI)
                         .thenRun(() -> UI.events.triggerEvent(new RepoOpenedEvent(repoId)))
+                        .thenRun(() -> UI.events.triggerEvent(new RepoListUpdatedEvent()))
```

Comment:
This keeps giving me
`Exception in thread "pool-4-thread-2" java.lang.IllegalStateException: This operation is permitted on the event thread only; currentThread = pool-4-thread-2`

Anything I can do about it?